osquery/tables/events/linux/bpf_file_events.cpp

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/logger/logger.h>
+#include <osquery/config/config.h>
+#include <osquery/registry/registry_factory.h>
+#include <osquery/sql/sql.h>
+#include <osquery/tables/events/linux/bpf_file_events.h>
+#include <osquery/filesystem/filesystem.h>
+#include <osquery/utils/system/uptime.h>
+
+#include <boost/algorithm/string.hpp>
+#include <iostream>
+#include <regex>
+
+namespace osquery {
+
+REGISTER(BPFFileEventSubscriber, "event_subscriber", "bpf_file_events");
+
+std::vector<std::string> monitored_paths;
+
+void configure_filtering() {
```

Comment:
Paths are expanded by the file_events implementation because osquery needs to install inotify watchers. In the case of BPF, this is not needed. Paths should be checked without performing expansion